packages/xfsprogs/xfsprogs.spec

```diff
@@ -0,0 +1,86 @@
+Name: %{_cross_os}xfsprogs
+Version: 5.15.0
+Release: 1%{?dist}
+Summary: Utilities for managing the XFS filesystem
+License: GPL+ and LGPLv2+
+URL: https://xfs.wiki.kernel.org
+Source0: http://kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-%{version}.tar.xz
+Patch1: 0001-libxfs-do-not-try-to-run-the-crc32selftest.patch
+Patch2: 0002-xfs_db-fix-a-complaint-about-a-printf-buffer-overrun.patch
+Patch3: 0003-xfs_db-fix-complaints-about-unsigned-char-casting.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libuuid-devel
+BuildRequires: %{_cross_os}libinih-devel
+BuildRequires: %{_cross_os}userspace-rcu-devel
+BuildRequires: %{_cross_os}libblkid-devel
+
+Requires: %{_cross_os}userspace-rcu
+Requires: %{_cross_os}libinih
+
+%description
+%{summary}.
+
+%package devel
+Summary: XFS filesystem-specific headers
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n xfsprogs-%{version} -p1
+
+%build
+%cross_configure \
+  --enable-editline=yes \
+  --enable-blkid=yes \
+  --enable-lto=no \
+  --enable-editline=no \
+  --enable-scrub=no
+
+
+%make_build \
+  INCLUDEDIR=%{_cross_includedir} \
+  LIBDIR=%{_cross_libdir}
```

Comment:
Are `INCLUDEDIR` and `LIBDIR` not set to the correct default values?

Reply:
These were lingering from other failures, they do not add anything.